### PR TITLE
fix invalid multiple flag import

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Fixes https://github.com/gtk-rs/glib/compare/master...GuillaumeGomez:regen

File: src/codegen/flags.rs [hunk 1]
```diff
@@ -58,9 +58,13 @@ pub fn generate(env: &Env, root_path: &Path, mod_rs: &mut Vec<String>) {
         try!(general::uses(w, env, &imports));
         try!(writeln!(w));
 
+        let mut first = true;
         for config in &configs {
             if let Type::Bitfield(ref flags) = *env.library.type_(config.type_id.unwrap()) {
-                mod_rs.push("\nmod flags;".into());
+                if first {
+                    mod_rs.push("\nmod flags;".into());
```
Comment (EPashkin) [inline src/codegen/flags.rs]: Just move it before for

Comment (GuillaumeGomez) [reply src/codegen/flags.rs]: Ah true, now the check is before... Good catch!

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by EPashkin** [comment]
Thanks.

**Pull request merged**